Mobile Buy SDK/Mobile Buy SDK/Additions/NSString+BUYAdditions.h

```diff
@@ -0,0 +1,129 @@
+//
+//  NSString+BUYAdditions.h
```

Comment:
It appears that the only 2 methods required are `buy_stringByAppendingBaseFileNameSuffix` and `buy_trim`